lg69t/README.md

```diff
@@ -66,7 +83,7 @@ To program the bootloader:
 3. Press and hold the `BOOT` button.
 4. Power on the module or release the `RESET` button.
 5. Release the `BOOT` button after the device is powered up.
-6. Run `stm32loader -p /dev/ttyUSB1 -e -w -v -a 0x08000000 quectel-bootloader-A.B.C.bin`
+6. Run `stm32loader -p /dev/ttyUSB1 -e -w -v -a 0x08000000 quectel-bootloader-1.0.4.bin`
```

Comment:
I don't like having an arbitrary version in there tbh. Results in copy/paste errors

Reply:
That's fair. The instructions above all note a specific UART and version number as an example, and then use them. I changed this one to be consistent.

I made the other examples that way so it was a little more clear how the numbering works and what UART to use, but I can just change that back.

Reply:
I changed the USBs to `/dev/ttyUSBn`, and changed all the version numbers back to `A.B.C` _except_ this one. This one I set to `X.Y.Z` to make it clear that it is not related to the software version number.